ckanext/benap/benap_schema.json

```diff
@@ -252,6 +251,7 @@
         "nl": "Naam van de organisatie [uitgever]",
         "de": "Name der Organisation  [Herausgeber]"
       },
+      "form_placeholder":"Great Library of Alexandria",
```

Comment:
Hier zou ik geen placeholder doen. "Great Library of Alexandria" kan verwarrend overkomen denk ik.